### ISSUE TITLE
Add Pulsar (pulsar-edit) as external editor

### ISSUE THREAD

**Issue by dsjosh** [desktop/desktop#16805]
### The feature request

Hello, could someone add pulsar (pulsar-edit) as one of the external editor options?

This is definitely possible since it is a spin-off of Atom editor, which is currently in the external editor list.

https://pulsar-edit.dev/

I saw another issue raised here with a similar request, but it was only resolved for the macos version.
https://github.com/desktop/desktop/issues/16231
I am unfortunately using Windows and requesting the Windows version of the feature.

Reference: https://github.com/desktop/desktop/blob/be4534ca7079c671b4d1381af089a4ccbba98d27/docs/technical/editor-integration.md
### Proposed solution

basically the same code for atom can be re-used. just need to refactor to suit pulsar's installation path and filename.

### Additional context

atom editor was sunsetted a few months ago and thus pulsar (pulsar-edit) was born from the fork of atom's code. everything is more or less the same between two editors hence the feasibility for pulsar is more or less guaranteed in the context of this feature request.

**Comment by steveward** [comment]
GitHub Desktop now provides support for [adding custom editors](https://docs.github.com/en/desktop/configuring-and-customizing-github-desktop/configuring-a-default-editor-in-github-desktop?platform=windows#configuring-a-custom-editor), so you can add Pulsar by following the steps in that linked help article. I'm going to close this out.
